tests/test_filters.py

```diff
@@ -63,6 +63,22 @@ def test_extract_pixels(test_filters: Filters) -> None:
     assert test_filters.images["pixels"].shape == (1024, 1024)
 
 
+@pytest.mark.parametrize("unit, x, y, expected", [
+    ('um', 100, 100, 97.65625),
+    ('nm', 50, 50, 0.048828125),
+    ])
+def test_extract_pixel_to_nm_scaling(test_filters_random: Filters, unit, x, y , expected) -> None:
+    """Test extraction of pixels to nanometer scaling."""
+    test_filters_random.images["extracted_channel"].size['real'] = {
+    'unit': unit,
+    'x': x,
+    'y': y
+    }
+    #test_filters_random.images["extracted_channel"].pxs()
```

Comment:
I think this can probably be removed since the PySPM method (`.pxs()`) is called within the `Filters.extract_pixel_to_nm_scaling()` method.

Reply:
Done and test still work